tests/services/order_validation_service/workflows/tomte/test_case_sample_rules.py

```diff
@@ -111,11 +112,51 @@ def test_sample_cycle_not_allowed(order_with_sample_cycle: TomteOrder):
     assert isinstance(errors[0], DescendantAsFatherError)
 
 
-def test_incest_is_allowed(order_with_siblings_as_parents: TomteOrder):
+def test_incest_is_allowed(order_with_siblings_as_parents: TomteOrder, base_store: Store):
     # GIVEN an order where parents are siblings
 
     # WHEN validating the order
-    errors: list[PedigreeError] = validate_pedigree(order_with_siblings_as_parents)
+    errors: list[PedigreeError] = validate_pedigree(
+        order=order_with_siblings_as_parents, store=base_store
+    )
 
     # THEN no error is returned
     assert not errors
+
+
+def test_existing_samples_in_tree(
```

Comment:
Not sure if this test adds any value - should it be removed?

Reply:
I agree. It could have some value if the new sample would have any family relation with the existing samples. But I would be ok with removing it